terraform/aws/analytical-platform-data-production/s3-glue-crawler/iam-policies.tf

```
@@ -0,0 +1,26 @@
+data "aws_iam_policy_document" "crawler_policy" {
```

```suggestion
data "aws_iam_policy_document" "alpha_vcms_data_crawler_policy" {
```

this needs scoping, incase we want to deploy more crawlers